server/hotrod/src/main/resources/hotrod.g

```diff
@@ -0,0 +1,343 @@
+// Hot Rod 2.x protocol
```

Comment:
The *.g file extension is picked up by antlr plugin and highlights the whole file as invalid. Can we use a different extension to avoid the clash?